src/libs/actions/IOU.js

```diff
@@ -624,9 +624,10 @@ function getMoneyRequestInformation(
     if (iouReport) {
         if (isPolicyExpenseChat) {
             iouReport = {...iouReport};
-
-            // Because of the Expense reports are stored as negative values, we substract the total from the amount
-            iouReport.total -= amount;
+            if (policy.outputCurrency === currency) {
```

Comment:
I think we better use lodash here so that it does not crash when we make a distance request in the policy because we are not passing any policy [here](https://github.com/Expensify/App/blob/990815f84567f39d4b84c45bf1952b306d0c4bfe/src/libs/actions/IOU.js#L799-L814) and accessing outputCurrency on it will crash the app.